README.md

```diff
@@ -91,6 +91,15 @@ DEM is available in the [PyPI repository](https://pypi.org/project/axem-dem/). I
 
 :information_source: The package name is axem-dem, but the command is `dem`.
 
+
+You can also install it with init-dem.sh from our repository. The installer script checks are you using the corresponding python version and install docker.io and axem-dem pip modul. Download it with the the following command:
```

Comment:
Rephrase to "The installer script checks if the corresponding python version is available and installs the docker and axem-dem pip modul.

(New line)
Download it with the following command:"

Reply:
fixed!